modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts

```diff
@@ -83,43 +83,30 @@ export async function syncSite(page) {
 		timeout: 100 * 1000,
 	});
 
-	// Known issue. See https://liferay.atlassian.net/browse/LRAC-13481
-
-	const tryAgainButton = page.getByRole('button', {name: 'Try Again'});
-
-	if (await tryAgainButton.isVisible()) {
-		await page.getByRole('button', {name: 'Previous'}).click();
-
-		await page.getByRole('button', {name: 'Next'}).click();
-	}
-
 	const wizard = page.getByTestId('VIEW_WIZARD_MODE');
 
 	await expect(wizard.getByText('Available Properties')).toBeVisible({
 		timeout: 100 * 1000,
 	});
 
-	await page
-		.getByTestId('Liferay DXP')
-		.getByRole('button', {name: 'Assign'})
-		.click();
+	const assignButton = await page.$(
+		'table.table tbody tr:first-child button'
+	);
+
+	await assignButton.click();
 
 	await page.getByRole('tab', {name: 'Sites'}).click();
 
-	await page
-		.getByTestId('1')
-		.getByTestId('Liferay DXP')
-		.getByLabel('')
-		.check();
+	await page.getByTestId('1').getByTestId('Liferay').getByLabel('').check();
```

Comment:
Hey @interaminense, I think this is still not right. You are considering that you will always have only one property created in your AC, but if more than one test is executed then you will have N properties created.

The way you did it, it will always be using the same property for all tests, so it probably won't fail in this case, but thinking about future test cases, you would have a single property that is collecting data from all your tests. The ideal here would be for each test to use the property that was created in the connection that the test made, so it would always be the first property in the list. What do you think?

Reply:
Sure thing, I'll update that.

Reply:
Done, please review it again.